app/views/posts/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Listing User Posts</h1>
```

Comment:
`app/views/posts/show.html.erb`

- Let's also display a placeholder text for the post-show page.